tests/update_reference_data.sh

```diff
@@ -54,14 +56,37 @@ for test_name in ${@:2}; do
       SPE9_CP_SHORT \
       EGRID INIT PRT SMSPEC UNRST UNSMRY
   fi
-done
-
-
 
+  if grep -q "norne_init" <<< $test_name
+  then
+    copyToReferenceDir \
+      $configuration/build-opm-simulators/tests/results/init/flow+norne/ \
+      $OPM_DATA_ROOT/norne/opm-simulation-reference/ \
+      NORNE_ATW2013 \
+      EGRID INIT
+  fi
+done
 
+if [ -z "${@:2}" ]
+then
+  # User did not specify tests to update, probe
+  pushd $OPM_DATA_ROOT > /dev/null
+  tests=""
+  git status | grep "SPE1CASE1" && tests="spe11"
+  git status | grep "SPE1CASE2" && tests="$tests spe12"
+  git status | grep "SPE3CASE1" && tests="$tests spe3"
+  git status | grep "SPE9_CP" && tests="$tests spe9"
+  git status | grep "NORNE_ATW2013" && tests="$tests norne_init"
+  popd > /dev/null
+fi
 
-echo -e "update reference data for ${@:2}\n" > /tmp/cmsg
-echo -e "Reason: fill in this\n" >> /tmp/cmsg
+echo -e "update reference data for $tests\n" > /tmp/cmsg
+if [ -z "$REASON" ]
+then
+  echo -e "Reason: fill in this\n" >> /tmp/cmsg
```

Comment:
this is for scripted generation of the commit message. update_reference_data.sh can be run interactive or scripted (through the trigger). the update_opm_data.sh script from opm-common generates a commit reason (line 46) that holds the links to the PR's specified. ref commit message here https://github.com/OPM/opm-data/pull/175/commits/f66766b6e25d98e87535a23702f6d2244b75fdd1